Support testcafe@2

Tescafé jsut released a new major version, which means this package has to update its peer dependency on it:
```json
"peerDependencies": {
    "testcafe": "~1"
  },
```
```
npm ERR! Could not resolve dependency:
npm ERR! peer testcafe@"~1" from @percy/testcafe@1.0.2
npm ERR! node_modules/@percy/testcafe
npm ERR!   dev @percy/testcafe@"1.0.2" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: testcafe@1.20.1
npm ERR! node_modules/testcafe
npm ERR!   peer testcafe@"~1" from @percy/testcafe@1.0.2
npm ERR!   node_modules/@percy/testcafe
npm ERR!     dev @percy/testcafe@"1.0.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
     "@percy/sdk-utils": "^1.1.1"
   },
   "peerDependencies": {
-    "testcafe": "~1"
+    "testcafe": ">=1"
   },
   "devDependencies": {
     "@percy/cli": "^1.9.1",

